config/application.rb

```diff
@@ -47,18 +48,30 @@ class Application < Rails::Application
     config.sandbox_email_mode = true
 
     config.lograge.enabled = true
+    config.lograge.logger = ActiveSupport::Logger.new(STDOUT)
+    config.lograge.formatter = Lograge::Formatters::Json.new
+    config.lograge.base_controller_class = ['ActionController::Base']
     config.lograge.custom_options = ->(event) do
       {
+        request_time: Time.current,
+        application: Rails.application.class,
         server_protocol: event.payload[:server_protocol],
-        forwarded_for: event.payload[:forwarded_for],
```

Comment:
@andyzito, is it possible this is needed to get the actual IP address?  I have a recollection we needed something special, but maybe it was only necessary before we moved to ECS.

Reply:
This seems to be handled with `x_forwarded_for` and appears both are `nil` in our current deployment setup.